src/main/java/util/InputNumber.java

```diff
@@ -0,0 +1,58 @@
+package util;
+
+import exception.DuplicateNumberException;
+import exception.NotOnlyNumberException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import static util.Constant.*;
+
+/**
+ * 사용자가 입력한 숫자에대해 다루는 클래스
+ */
+public class InputNumber {
+    private final Scanner sc= new Scanner(System.in);
+    private final ArrayList<Integer> inputNumArray= new ArrayList<>();
+
+    public ArrayList<Integer> getInputNumArray() {
+        return inputNumArray;
+    }
+
+    public char[] inputNumberScanner() {
+        this.inputNumArray.clear();
+        String scStr = sc.nextLine();
+        return scStr.toCharArray();
+    }
+
+    public void inputNumberValidation(char[] scChar) {
+        ArrayIndexExceptionValidation(scChar);
+        for (char c : scChar) {
+            notOnlyExceptionValidation(c);
+            DuplicateNumberExceptionValidation(c);
+            this.inputNumArray.add(c - CHAR_TO_INT);
+        }
+    }
+
+    private void DuplicateNumberExceptionValidation(char c) {
```

Comment:
메서드 이름에서 첫글자는 항상 소문자여야 합니다. ☺️

<img width="980" alt="스크린샷 2022-05-29 오후 10 30 02" src="https://user-images.githubusercontent.com/92219795/170871391-39703fd4-e83a-4e65-806e-0559d1d54eca.png">

```suggestion
    private void duplicateNumberExceptionValidation(char c) {
```